backend/src/main/java/asegroup1/api/models/landregistry/LandRegistryQuerySelect.java

```diff
@@ -1,144 +1,151 @@
 package asegroup1.api.models.landregistry;
 
-import java.util.ArrayList;
-import java.util.EnumSet;
+import java.util.LinkedHashMap;
+
+import asegroup1.api.models.landregistry.LandRegistryQuery.Aggrigation;
+import asegroup1.api.models.landregistry.LandRegistryQuery.Selectable;
 
-/**
- * 
- * @author Rikkey Paal
- */
 public class LandRegistryQuerySelect {
 
-	private ArrayList<Selectable> selectableMap;
+	private LinkedHashMap<String, SelectObj> selectValues;
 
-	/**
-	 * Initialise the LandRegistryQuerySelect with specified {@link Selectable
-	 * selectable's}.
-	 * 
-	 * @param selectables to initialise the code
-	 */
 	public LandRegistryQuerySelect(Selectable... selectables) {
-		selectableMap = new ArrayList<Selectable>();
-		select(selectables);
+		selectValues = new LinkedHashMap<String, SelectObj>();
```

Comment:
Generics do not need to be specified in initialisation

Reply:
I know, but my ide gets a bit annoyed if they arent sometimes.

Reply:
If you want me to remove it then i will, but i dont see how it can cause any problems.